platform/mds/mds/src/main/java/org/motechproject/mds/dto/EntityDto.java

```diff
@@ -40,9 +42,11 @@
     private Set<String> readOnlySecurityMembers;
     private String superClass;
     private boolean abstractClass;
+    private boolean isDiscriminated;
     private boolean securityOptionsModified;
     private Integer maxFetchDepth;
     private boolean readOnlyAccess;
+    private List<FieldDto> fieldsAdded;
```

Comment:
I would extend the name, so that it tells from where were the fields added (fromParent? fromChild?)